Proposals/xgov-9.md

```diff
@@ -0,0 +1,44 @@
+---
+period: 9
+title: AWallet 
+author: Ludovit Scholtz (LudovitScholtz#9304) (@EAlgonaut)
+company_name: Scholtz & Co
+category: Wallets
+focus_area: Open source algorand wallets
+open_source: Yes
+amount_requested: 10,000A
+status: Draft
+---
+
+## Abstract
+AWallet is [first open source](https://forum.algorand.org/t/algorand-wallet-open-source/3497) Algorand wallet. Wallet allows multisig features, ledger powered accounts, rekeying, making account online and protect algorand network, vanity account generation, ARC14 support, contains algorand native payment gateway, and more. AWallet is built as docker image so that anybody can run it locally in their environment and debug for example sandbox accounts. 
```

Comment:
```suggestion
AWallet is <a href="https://forum.algorand.org/t/algorand-wallet-open-source/3497">first open source</a> Algorand wallet. Wallet allows multisig features, ledger powered accounts, rekeying, making account online and protect algorand network, vanity account generation, ARC14 support, contains algorand native payment gateway, and more. AWallet is built as docker image so that anybody can run it locally in their environment and debug for example sandbox accounts. 
```